src/test/common/test_lockdep.cc

```diff
@@ -57,6 +58,7 @@ int main(int argc, char **argv) {
        << std::endl;
   exit(0);
 #endif
+  PrCtl unset_dumpable;
```

Comment:
could you move this statement right before where coredump is created? i.e., you could move it to where `EXPECT_DEATH()` gets called.